data/registry/instrumentation-js-fastify-otel.yml

```
@@ -0,0 +1,20 @@
+# cSpell:ignore fastify
+title: Fastify Instrumentation Library
+registryType: instrumentation
+language: js
+tags:
+  - Node.js
+  - fastify
+license: MIT
+description: Fastify OpenTelemetry auto-instrumentation library.
+authors:
+  - name: Fastify
+     url: https://github.com/fastify
+urls:
+  repo: https://github.com/fastify/otel
+createdAt: 2025-01-27
+package:
+  registry: npm
+  name: '@fastify/otel'
+  version: 0.1.0
+isFirstParty: false
```

If I understand this flag correctly, it's `true` since `fastify` authors provide the `fastify` instrumentation now. 🙂 

```suggestion
isFirstParty: true
```

---

Ah yes! Thanks for catching that